tests/e2e/test_api.py

```diff
@@ -74,8 +77,24 @@ def setup_module(module):
         else:
             print("Setting up for standalone test execution\n")
 
+        # Determine the hostname for the OLS route
+        ols_url = os.popen("oc get route ols -o jsonpath='{.spec.host}'").read().strip()
```

Comment:
we've got some helper functions for interacting with `oc` under "cluster.py", please take a look at that so we can do this in a consistent way

Reply:
it might be saver to use `subprocess.Popen` with all the error checking etc.